api.js

```diff
@@ -18,7 +17,7 @@ function Api(files, options) {
 
 	EventEmitter.call(this);
 
-	assign(this, options);
```

Comment:
You can't just remove this. The options are used directly on `this` in the code and needs to be changed to use `this.options.x`.